lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -515,14 +644,40 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
         myLayer = myLayer[0];
         if (myLayer && myLayer.Style && myLayer.Style.length > 0) {
           for (var j = 0; j < myLayer.Style.length; ++j) {
-            if (!defined(this.isNcWMS) && myLayer.Style[j].Name && myLayer.Style[j].Name.match(/boxfill\/rainbow/i)) {
+            if (!defined(this.isNcWMS) && myLayer.Style[j].Name &&
+                (myLayer.Style[j].Name.match(/boxfill\/rainbow/i) ||
+                 myLayer.Style[j].Name.match(/default-scalar\/default/i) ||
+                 myLayer.Style[j].Name.match(/default-vector\/default/i))) {
```

Comment:
Are these standards-defined style names that clearly mark the server out as NcWMS? Could you put in a ref to the doc maybe?